### ISSUE TITLE
WebM in Photoshop ?

### ISSUE THREAD

**Issue by SheepDomination** [fnordware/AdobeWebM#23]
I can't find the plugin; searched everywhere within PhotoshopCC-2015 ?

**Comment by fnordware** [comment]
As far as I know, there's no SDK support for reading and writing movies into/out of Photoshop. Seems like supporting  MediaCore would be a good way for Adobe to do it.

What happens when you load your Photoshop movie file into Media Encoder, Premiere, or After Effects?

**Comment by SheepDomination** [comment]
I want to export an image to WebM, I can't find where the WebM plugin is located in Photoshop ?

**Comment by fnordware** [comment]
There is no WebM plug-in for Photoshop, and I don't think there is a way to write one. Photoshop can work with movies now, but I don't think I can write a plug-in to support new movie formats.

As part of this project, I have a Photoshop plug-in for the WebP still image format. It's related to WebM, but it's a different format.

If you have a Photoshop animation and you want to make a WebM movie, maybe trying opening the Photoshop document in Media Encoder, Premiere, or After Effects and then making a WebM from there?

**Comment by SheepDomination** [comment]
I have the WebM for still Image plugin for Photoshop, where in Photoshop do I find it, it's not in the export menu, it's not in the filters menu ?

**Comment by fnordware** [comment]
To use the [WebP](https://developers.google.com/speed/webp/?csw=1) plug-in, go to File > Save As and choose WebP as the format.

You can also open WebP files using File > Open.

**Comment by SheepDomination** [comment]
WebP is not listed as the format in the File > Save As list ? I assume since the version I'm using was updated in Maya of this year (2015) that it is compatible with PhotoshopCC-2015 ?

**Comment by fnordware** [comment]
Should work in the latest Photohsop. Copy WebP.8bi to the Photoshop plug-ins folder.

**Comment by SheepDomination** [comment]
I have the latest version of the 8bi file in the Photoshop plug-ins folder, it's not showing up ?

**Comment by fnordware** [comment]
If you go to Help > About Plug-In, is WebP listed? You can also look in Help > System Info to see if there's more information.

**Comment by SheepDomination** [comment]
Help > About Plug-in does list the WebP plugin, I don't understand why Save As is not listing it, odd.

**Comment by fnordware** [comment]
WebP will only be an option if you have an 8-bit RGB file.

Maybe first try downloading a WebP [sample](https://developers.google.com/speed/webp/gallery2) and open it. Then try saving it.

**Comment by SheepDomination** [comment]
Ah, I thought it would work with 16-bit images.  It works, although there are no options, or does WebP not allow options like the PNG format ?

One other thing, your OpenEXR plug-in for photoshop, you wouldn't happen to offer a discount on purchasing ?

**Comment by fnordware** [comment]
Nope, WebP is for the web, which is generally 8-bit. You should see an options dialog after you enter the file name though!

**Comment by SheepDomination** [comment]
Thanks it works.

I assume no discount on OpenEXR ?

**Comment by fnordware** [comment]
I already give everyone the discount! ;) It used to be $95.

There's the free ProEXR EZ if you don't need layers.

**Comment by SheepDomination** [comment]
What do you mean it use to be $95; I didn't know about such discount ? How much was the discount ?

**Comment by fnordware** [comment]
Used to be $95, now it's $75.

This is not really the place to be discussing ProEXR.

**Comment by SheepDomination** [comment]
Could I email you ?
